vaadin-tabs-flow-parent/vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/Tabs.java

```diff
@@ -99,7 +99,9 @@ public Tabs(Tab... tabs) {
 
     /**
      * Constructs a new object enclosing the given autoselect option and tabs,
-     * with {@link Orientation#HORIZONTAL HORIZONTAL} orientation.
+     * with {@link Orientation#HORIZONTAL HORIZONTAL} orientation. If this
+     * constructor is used, any selection listener added afterwards will not be
+     * invoked with a probable autoselect here.
```

Comment:
```suggestion
     * Constructs a new object enclosing the given autoselect option and tabs,
     * with {@link Orientation#HORIZONTAL HORIZONTAL} orientation.
     * <p>
     * Unless auto-select is disabled, the first added {@link Tab} component
     * will be automatically selected. Any selection change listener added
     * afterwards will not be notified about the auto-selected tab.
```
Could be phrased a bit more clearer IMO.

Note that the more important constructor is the one above, which has the same ambiguous phrasing as the `add` method. It should also be improved to something like this:
```
     * Constructs a new object enclosing the given tabs, with
     * {@link Orientation#HORIZONTAL HORIZONTAL} orientation.
     * <p>
     * The first added {@link Tab} component will be automatically selected. Any
     * selection change listener added afterwards will not be notified about the
     * auto-selected tab.
```

Reply:
Could not directly commit here, addressed in the latest commit.